core/modules/telemetry/telemetry.telemetry.inc

```diff
@@ -44,7 +44,13 @@ function telemetry_telemetry_data($key) {
     case 'php_version':
       return PHP_MAJOR_VERSION . '.' . PHP_MINOR_VERSION . '.' . PHP_RELEASE_VERSION;
     case 'mysql_version':
-      return Database::getConnection()->databaseType() === 'mysql' ? Database::getConnection()->version() : NULL;
+      // Trim the version number to only major + minor, otherwise we'd flood the
+      // table with dozens of bug fix versions.
```

Comment:
@jenlampton I'm not sure if we should be doing this. If we ever need to troubleshoot issues that happen in specific bug fix versions, then we won't have the data. The borg side of our telemetry implementation already allows us to choose to aggregate on either 0/1/2 decimals for the versions:
<img width="1238" alt="image" src="https://github.com/backdrop/backdrop/assets/2423362/45a254c8-8921-4762-89ac-c0c132d7f611">

Also, if you take a look at the `case 'php_version':` above, you'll see that we aren't trimming the release (bug fix) version there either.

Point being: lets collect more accurate data in case we ever need it - the granularity on display on the metrics table can easily be limited to only major/minor.

Reply:
I agree, I don't think this trimming is necessary. It's trimmed on the display side like @klonos says above.